Content.Client/Fax/System/FaxVisualsSystem.cs

```diff
@@ -0,0 +1,33 @@
+using Robust.Client.GameObjects;
+using Content.Shared.Fax.Components;
+using Content.Shared.Fax;
+
+namespace Content.Client.Fax.System
+{
```

Comment:
File-scoped namespaces.

Reply:
Maybe i am a bit confused on what file-scoped namespaces means. I assumed the namespace for faxvisualsystem should be in client.fax.system?

Reply:
i changed the faxvisualssystem to just faxvisuals then put it in a folder called faxvisuals.

Reply:
A file-scoped namespace is a C# syntax thing that makes the namespace apply to the entire file, rather than having to wrap everything in another set of curly brackets.

Just replace 
```c#
namespace Content.Client.Fax.System
{
    // Everything
}
```

With
```c#
namespace Content.Client.Fax.System;

/// Everything
```

Reply:
See here should explain more https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-10.0/file-scoped-namespaces